pkg/client/client.go

```diff
@@ -98,55 +99,83 @@ func (c *Client) agent() string {
 	return fmt.Sprintf("helm/%s", version.Version)
 }
 
-// CallService is a low-level function for making an API call.
-//
-// This calls the service and then unmarshals the returned data into dest.
-func (c *Client) CallService(path, method, action string, dest interface{}, reader io.Reader) error {
-	u, err := c.url(path)
-	if err != nil {
-		return err
-	}
+// Get calls GET on an endpoint and decodes the response
```

Comment:
Tiny nit: It's not clear from the comment that it decodes the response into `v`.